### PR TITLE
Add book section

### PR DESCRIPTION
**Pull request by Charismatron**
Would you be willing to add a book section to your PWA resource article?

Thanks for your consideration.

### CODE REVIEW

File: src/articles/_posts/2016-11-11-pwa-resources.md [hunk 1]
```diff
@@ -92,3 +92,10 @@ Take a look at what others are doing with Progressive Web Apps. Get inspiration
 - [Google PWA Showcase](https://developers.google.com/web/showcase/): Google’s showcase of companies which have befitted from PWAs.
 - [The Business Case for Progressive Web Apps](https://cloudfour.com/thinks/the-business-case-for-progressive-web-apps/): Jason Grigsby writes a compelling case for the business benefits of PWAs.
 - [PWAs: An African Perspective](https://dev.opera.com/articles/pwa-nigeria-kenya-interview/): We interview two great developers from Nigeria and Kenya about PWAs. Read their thoughts on how PWAs can help solve problems in that region.
+
+## Books
+
+Books to help you learn about PWAs
```
Comment (fsoder) [inline src/articles/_posts/2016-11-11-pwa-resources.md]: End with full stop?

File: src/articles/_posts/2016-11-11-pwa-resources.md [hunk 2]
```diff
@@ -92,3 +92,10 @@ Take a look at what others are doing with Progressive Web Apps. Get inspiration
...
+
+- [Progressive Web Apps](https://www.manning.com/books/progressive-web-apps): Progressive Web Apps teaches you PWA design and the skills you need to build fast, reliable websites by taking you step-by-step through real world examples in this practical tutorial.
```
Comment (fsoder) [inline src/articles/_posts/2016-11-11-pwa-resources.md]: Maybe avoid repeating the title, so:

...: This book teaches you...

The "in this practical tutorial" at the end seems like a disconnected appendix.

**Comment by Charismatron** [comment]
I agree with you suggested changes to the title header and to the book’s
description. So it would say: This book teaches you PWA design and the
skills you need to build fast, reliable websites by taking you step-by-step
through real world examples.

Cheers and thanks for agreeing to add it!

**Comment by fsoder** [comment]
Yes, that sounds good to me, will you push a fixup commit?

**Comment by fsoder** [comment]
I made the edits, if I messed something up, please send a fixup. I'll merge this in a moment.

**Comment by Charismatron** [comment]
Thanks! I would have fixed it myself, but I was out of the house buying a
Christmas tree.

Happy holidays

**Pull request merged**